File: EffortEstimator/ClientApp/src/app/objects/conference-entity.ts
export class ConferenceEntity {
    conferenceId: number;
    groupName: string;
    topic: string;
    description: string;
    startDate: string;

    constructor(conferenceId: number, groupName: string, topic: string, description: string, startDate: string) {
        this.conferenceId = conferenceId;
        this.groupName = groupName;
        this.topic = topic;
        this.description = description;
        this.startDate = startDate;
    }
}

File: EffortEstimator/Models/ConferenceEntity.cs
using System;

namespace EffortEstimator.Models
{
    public class ConferenceEntity
    {
        public int ConferenceId { get; set; }
        public string GroupName { get; set; }
        public string Topic { get; set; }
        public DateTime StartDate { get; set; }
        public string Description { get; set; }
    }
}
